test/runtests.jl

```diff
@@ -7,6 +7,9 @@ using Transforms: _try_copy, _periodic
 using Test
 using TimeZones
 
+# TODO - this is needed to make mapslices work with AxisArrays - should be removed.
+AxisArrays.reduced_indices(X::Tuple{Vararg{Axis}}, r::UnitRange) = AxisArrays.reduced_indices(X, Tuple(collect(r)))
```

Comment:
I hate this. I think this should just be added to AxisArrays but depending how difficult that is to get in it might just be easier to ditch AxisArrays altoghther.

Reply:
https://github.com/JuliaArrays/AxisArrays.jl/pull/194